ui/user/src/lib/components/edit/ThreadModelSelector.svelte

```diff
@@ -118,26 +118,56 @@
 		isUpdatingModel = true;
 
 		try {
-			const updatedThread = await updateThread(project.assistantID, project.id, {
-				...threadType,
-				model: model,
-				modelProvider: provider
-			});
+			let retryCount = 0;
+			const maxRetries = 5;
+
+			while (retryCount < maxRetries) {
```

Comment:
pico-nit


```suggestion
			const maxTries = 5;

			while (retryCount < maxTries) {
```

(this variable is also referenced farther down)